cogs5e/funcs/scripting/evaluators.py

```diff
@@ -158,6 +158,11 @@ def create_cc_nx(name: str, minVal: str = None, maxVal: str = None, reset: str =
                 character.consumables.append(new_consumable)
                 self.character_changed = True
 
+        def create_cc(name: str, **kwargs):
+            if name in set(con.name for con in character.consumables):
```

Comment:
perhaps use `if cc_exists(name)`?

Reply:
Oh yeah, that certainly would be more elegant. Fixed in latest commit.
Also, shouldn't create_cc_nx use `not cc_exists(name)`?